google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -239,8 +276,54 @@ def run(self):
             self.model.set_done(item_counter)
         finally:
             self.session.commit()
+            self.session.autocommit = autocommit
             self.session.autoflush = autoflush
 
+    @staticmethod
+    def model_action_wrapper(session,
+                             inventory_iterable,
+                             pre_action,
+                             action,
+                             post_action,
+                             flush_count,
+                             is_resource=False):
+        """Model action wrapper.
+
```

Comment:
Please add a comment here to describe why this `model_action_wrapper` is needed, i.e. reduce repetitive code usage and checks.